src/Timer.js

```diff
@@ -58,16 +90,43 @@ class Timer extends React.Component {
   }
 
   render () {
-    const { time, seconds, on } = this.state
-    const elapsed = hhmmss(seconds)
+    const { time, seconds, on, directionMode, countdownTime } = this.state
+    const elapsed = directionMode === direction.normal
+      ? hhmmss(seconds)
+      : countdownTime - seconds >= 0
+        ? hhmmss(countdownTime - seconds)
+        : (
+          <span style={{color: "#600"}}>
+            - {hhmmss(seconds - countdownTime)}
+          </span>
+        )
 
     return (
       <Flex css={{ alignItems: 'center' }}>
+        {!on && (
+          <Button
+            bg={directionMode === direction.normal ? '#666' : '#060'}
+            onClick={this.toggleDirection}> switch to {' '}
+            {directionMode === direction.normal
+              ? 'countdown mode'
+              : 'normal mode'
+            }
+          </Button>
+        )}
         {!on && seconds > 0 && (
           <Button mr={1} onClick={this.reset}>
             reset
           </Button>
         )}
+
+        {directionMode === direction.countdown && (
+          !on
+            ? <input
+                value={countdownTime/60}
+                onChange={this.updateCountdownTime}
+              />
+            : <Mono px={2}>Presentation Time: {countdownTime/60} minutes</Mono>
```

Comment:
Might be nice if the input and the running countdown are the same piece of UI – i.e. when it's not running, it's editable as an input; and when it's running it's readonly – would have to make a decision about what happens on pause in that case, but just spitballing here

Reply:
Yes, this would be actually excellent, making it read-only when running would make a lot of sense.

Reply:
We could also use an actual toggle button for switching between the modes.